src/hooks/useEtherspotHistory.ts

```diff
@@ -59,24 +64,47 @@ const useEtherspotHistory = (chainId?: number): IEtherspotHistoryHook => {
 
   const getAccountTransaction = async (
     hash: string,
-    historyChainId: number = defaultChainId,
+    historyChainId: number = defaultChainId
   ): Promise<Transaction | undefined> => {
     try {
       const dataService = getDataService();
+      console.log('dataservice getAccountTransaction', dataService);
       return dataService.getTransaction({ hash, chainId: +historyChainId });
     } catch (e) {
       console.warn(
-        `Sorry, an error occurred whilst trying to fetch the transaction`
-        + ` for hash ${hash}. Please try again. Error:`,
-        e,
+        `Sorry, an error occurred whilst trying to fetch the transaction` +
+        ` for hash ${hash}. Please try again. Error:`,
+        e
       );
     }
-  }
+  };
 
-  return ({
+  const getAccountTransactionStatus = async (
+    fromChainId: number,
+    toChainId: number,
+    hash: string,
+    provider?: BridgingProvider
+  ): Promise<TransactionStatus | undefined> => {
+    try {
+      const dataService = getDataService();
+      return dataService.getTransactionStatus({ fromChainId, toChainId, transactionHash: hash, provider });
+    } catch (e) {
+      console.error(e);
```

Comment:
```suggestion
```
Sorry i also added this to try and tease out the error when we were debugging this earlier!